pages/about.vue

```diff
@@ -10,14 +10,14 @@
                     <p>
                         We are cdnjs, the #1 free and open source CDN built to make life easier for developers
                         serving <a :href="utm('https://w3techs.com/technologies/details/cd-cdnjs', 'about')">
-                            over 12.5% of all websites</a>, powered by Cloudflare.
+                            over 13.0% of all websites</a>, powering 46.3% of all websites that use a JavaScript CDN, and powered by Cloudflare.
```

Comment:
```suggestion
                            over 13.0% of all websites</a>, and powered by Cloudflare.
```
I don't think this extra copy is needed